src/github.com/getlantern/lantern-mobile/Dockerfile

```diff
@@ -5,7 +5,7 @@
 FROM fedora:22
 MAINTAINER "Ulysses Aalto" <uaalto@getlantern.org>
 
-ENV GO_VERSION go1.5.1
+ENV GO_VERSION go1.5.2
```

Comment:
Seems we can safely remove these `ENV` definitions to avoid invalidate intermediate docker images too often. They are not used until redefined.